lib/rules/block-indentation.js

```diff
@@ -83,16 +85,46 @@ module.exports = class BlockIndentation extends Rule {
 
     switch (configType) {
       case 'number':
-        return config;
+        return {
+          indentation: config,
+        };
       case 'boolean':
-        return config ? 2 : false;
+        if (config) {
+          return {
+            indentation: 2,
+          };
```

Comment:
Can you store this in a `DEFAULT_CONFIG` object?

Reply:
Done. I wasn't sure if `ignoreComments` should also be specified in the `DEFAULT_CONFIG`